zebrad/src/components/tracing/component.rs

```diff
@@ -266,7 +269,19 @@ impl Tracing {
             info!("activated progress bar");
 
             if config.log_file.is_some() {
-                eprintln!("waiting for initial progress reports...");
+                // Clear screen
+                eprint!("\x1B[2J");
+                {
+                    eprintln!(
+                        "{}",
+                        std::str::from_utf8(&ZEBRA_ART)
+                            .expect("should always work on a UTF-8 encoded constant")
+                    );
+                }
+                eprintln!("Thank you for running a zebrad {} node!", app_version());
```

Comment:
> * Whether to mention if it's mainnet or testnet like zcashd does (we don't have this information easily available where the printing happens)

Let's pass `cfg_ref.network.network` to the `Tracing::new()` method, then print `network.lowercase_name()` as part of this message?

Reply:
done in cacac5ac

Reply:
Thanks!